### PR TITLE
V3 world coords

### PR DESCRIPTION
**Pull request by votan73**
Handles those issues:
https://github.com/sirinsidiator/ESO-LibGPS/issues/2
https://github.com/sirinsidiator/ESO-LibGPS/issues/3 (partial)
https://github.com/sirinsidiator/ESO-LibGPS/issues/5
https://github.com/sirinsidiator/ESO-LibGPS/issues/6

### CODE REVIEW

File: src/compatibility.lua [hunk 1]
```diff
@@ -2,11 +2,10 @@
 -- Distributed under The Artistic License 2.0 (see LICENSE)     --
 ------------------------------------------------------------------
 
-if not LibStub then return end
-
-local MAJOR, MINOR = "LibGPS2", 18
-local lib = LibStub:NewLibrary(MAJOR, MINOR)
+local MAJOR, MINOR = "LibGPS2", 19
```
Comment (sirinsidiator) [inline src/compatibility.lua]: minor version should be set to 22 here

File: src/compatibility.lua [hunk 2]
```diff
@@ -2,11 +2,10 @@
...
+local lib = LibStub and LibStub:NewLibrary(MAJOR, MINOR)
 assert(lib, "LibGPS2 compatibility layer was loaded more than once. Please ensure that its files are not included from other addons.")
+lib = lib or {}
```
Comment (sirinsidiator) [inline src/compatibility.lua:8]: This line won't be executed as the assertion before will fail when lib is nil and throw an error

File: src/WaypointManager.lua [hunk 3]
```diff
@@ -59,11 +69,22 @@ function WaypointManager:RemovePlayerWaypoint()
...
 
-function WaypointManager:SetMeasurementWaypoint(x, y)
+function WaypointManager:SetMeasurementWaypoint(localX, localY)
     -- this waypoint stays invisible for others
     self.suppressCount = self.suppressCount + 1
     self.LMP:SuppressPing(MAP_PIN_TYPE_PLAYER_WAYPOINT)
-    self:SetPlayerWaypoint(x, y)
+
+    local x, y = 1, 1
+    if(math.abs(pwx - x) < 10000) then x = pwx + 10000 end
```
Comment (sirinsidiator) [inline src/WaypointManager.lua]: this throws an error "operator - is not supported for nil - number". Looks like pwx and pwy should be changed to localX and localY?

**Review by sirinsidiator — CHANGES_REQUESTED** [review]
Are you sure that's the most current version of your code? I get recurring errors when I run this version with other addons enabled. See comments in code.

File: src/WaypointManager.lua (hunk 3 above)
Comment (votan73) [reply src/WaypointManager.lua]: Yes and no. This does not work in zones where the position of the player is "stuck". A fix point far away 10000,10000 works.

**Pull request merged**